frontend/packages/dev-console/integration-tests/features/addFlow/create-from-docker-file.feature

```diff
@@ -10,33 +10,35 @@ Feature: Create Application from Docker file
 
         @regression
         Scenario: Dockerfile details after entering git repo url: A-05-TC01
-            Given user is on Import from Docker file page
-             When user enters docker git url as "https://github.com/sclorg/nodejs-ex.git"
-             Then git url "https://github.com/sclorg/nodejs-ex.git" gets Validated
-              And application name displays as "nodejs-ex-git-app"
-              And name field auto populates with value "nodejs-ex-git" in Import from Docker file page
+            Given user is on Import from Git form
+             When user enters Git Repo URL as "https://github.com/docker/labs/tree/master/beginner/flask-app"
```

Comment:
Do not use this git repo url. It is not a valid repo url. You need to enter `https://github.com/docker/labs` and then add context directory for this to correctly work. For this test I would suggest suing `https://github.com/rohitkrai03/flask-dockerfile-example`

Reply:
Updated